### PR TITLE
Fix to manage properly the terraform stdout content in the progress bar

### PR DESCRIPTION
**Pull request by arbulu89**
Fix to manage properly the terraform stdout content.

- In the case of an error, the `content` was set with the error message, so it was not getting the correct data to create the progress information. Besides that, the text for failed scenario was not set.

- Rarely, the terraform stdout was coming with `nil` value. The `blank` method adds a safe guard to that scenario, returning an empty progress information.

### CODE REVIEW

File: app/controllers/deploys_controller.rb [hunk 1]
```diff
@@ -76,9 +79,13 @@ def write_output(content, success)
 
   def update_terraform_progress(content, error)
     progress = {}
+    return progress if content.blank?
+
     created_resources = content.scan(/Creation complete after/).size
     planned_resources_count = KeyValue.get(:planned_resources_count)
-    text = if created_resources == planned_resources_count
+    text = if !error.nil?
```
Comment (MalloZup) [inline app/controllers/deploys_controller.rb:86]: `unless` is the ruby semantic

Comment (arbulu89) [reply app/controllers/deploys_controller.rb:86]: `unless` cannot be used here as we have a `if/elsif/else` statement.

Comment (bear454) [reply app/controllers/deploys_controller.rb:86]: Consider `if error.present?`

**Review by bear454 — APPROVED** [review]

File: app/controllers/deploys_controller.rb (hunk 1 above)
Comment (MalloZup) [reply app/controllers/deploys_controller.rb:86]: I would prefer also something without the inverse logic as suggested by @bear454 .

Otherwise looks good

Comment (arbulu89) [reply app/controllers/deploys_controller.rb]: Changed!
I have added some new `expects` in the UTs to check this scenario

**Pull request merged**